### PR TITLE
[ChangeLog] SE-0326 & style fixes

### PR DESCRIPTION
**Pull request by stevapple**
- Add a ChangeLog entry for SE-0326, which is implemented in Swift 5.7;
- Fix the style of some other changes.

Resolves SR-NNNN.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0326][]:
+
+Parameter and result types can now be inferred from from the body of multi-statement closures.
```
Comment (benrimmington) [inline CHANGELOG.md:10]:
```suggestion
Parameter and result types can now be inferred from the body of multi-statement closures.
```

Comment (benrimmington) [reply CHANGELOG.md:10]: These lines should also be indented by two spaces, to become part of the list item.

(The last few changelog entries have the same problem.)

Comment (stevapple) [reply CHANGELOG.md]: @benrimmington I updated the style for others too:)

File: CHANGELOG.md [hunk 2]
```diff
@@ -157,29 +163,32 @@ For example, Swift 5.6 would allow the following code, which at runtime would co
     return [ 1: "One", 2: "Two" ]
   }
   ```
+
+**Add new entries to the top of this section, not here!**
+
 Swift 5.6
 ---------
 
 * [SE-0327][]:
```
Comment (benrimmington) [inline CHANGELOG.md:172]: Please could you also add this release date:

```suggestion
Swift 5.6
---------

### 2022-03-14 (Xcode 13.3)

* [SE-0327][]:
```

**Review by benrimmington — CHANGES_REQUESTED** [review]
Thanks! I reviewed your changes with the "[Hide whitespace][]" option.

[Hide whitespace]: <https://github.com/apple/swift/pull/41852/files?diff=unified&w=1>

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,88 +5,94 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0326][]:
+
+  Parameter and result types can now be inferred from the body of multi-statement closures.
```
Comment (xedin) [inline CHANGELOG.md:10]: I think the first sentence should start from something like `Type-checking behavior of multi-statement closures has been unified with that of single-expression closures.`

**Review by xedin — CHANGES_REQUESTED** [review]
Please move styling changes into a separate PR.

**Comment by xedin** [comment]
My PR has been merged, so closing this one.

**Pull request closed without merging**